README-DEVELOP.md

```diff
@@ -22,18 +21,6 @@ You need to have installed
 There are a number of nodejs packages are needed, like [typescript](https://www.typescriptlang.org/), but you can get those via `npm`,
 which is described in a below [section](#how-to-run-code-in-this-github-repository).
 
-
-## Dependency hell / Current Issues
-There is a dependency on isomorphic-fetch
-if having issues compiling, replace the index.d.ts file with this
-
-```typescript
-declare namespace _fetch { }
-declare const _fetch: any;
-export = _fetch;
-```
```

Comment:
@juanfranblanco I don't add those codes. as I said before, `isomorphic-fetch` can be removed